slack-moderate-words/model/filter.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package model
+
+type FilterConfig []struct {
+	Triggers []string `yaml:"triggers"`
+	Action   string   `yaml:"action"`
```

Comment:
(Not in this PR) but it'd be nice to validate the filter config as a follow up. Example - validate that `action` is a valid slack action, etc.

Reply:
That sounds a good one :)
will do